router/pods_test.go

```diff
@@ -800,30 +800,26 @@ func TestGetHealthCheckFromPodPort(t *testing.T) {
 
 // TestValidatePath tests the internal function to validate a proper path used in annotations
 func TestValidatePath(t *testing.T) {
-	testNoPrefix := "test"
-	testPathFail := "/test/%2a/%"
-	testPathPass := "/test/%2a/aa/a"
-	testPathPassProperEncoding := "/test/par%2ate/aa/a"
-	testPathFailInvalidEncoding := "/test/hello%zzworld/aa/a"
```

Comment:
Yes, as those are still valid nginx paths, though probably not useful. Nginx will take anything that is a valid pcre regex syntax. Therefor our test cases is basically just anything that fails to compile a regex in go.